### PR TITLE
Remove initialAspectRatio

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Since we don't currently support locking the aspect ratio, having `initialAspectRatio` as an option doesn't really make sense, so we should remove it.

Spec PR: https://github.com/WICG/document-picture-in-picture/pull/36
Chromium CL: https://chromium-review.googlesource.com/c/chromium/src/+/4518212

FYI @steimelchrome @liberato-at-chromium

### CODE REVIEW

File: site/en/docs/web-platform/document-picture-in-picture/index.md [hunk 1]
```diff
@@ -130,33 +127,24 @@ pipButton.addEventListener('click', async () => {
 
 ### Set the size of the Picture-in-Picture window
 
-To set an aspect ratio, set the `initialAspectRatio` option of `documentPictureInPicture.requestWindow()` to the desired Picture-in-Picture window aspect ratio.
+To set the size of the Pictire-in-Picture window, set the `width` and `height` options of `documentPictureInPicture.requestWindow()` to the desired Picture-in-Picture window size. Chrome may clamp those options values if they are too large or too small to fit a user-friendly window size.
```
Comment (rachelandrew) [inline site/en/docs/web-platform/document-picture-in-picture/index.md:130]:
```suggestion
To set the size of the Picture-in-Picture window, set the `width` and `height` options of `documentPictureInPicture.requestWindow()` to the desired Picture-in-Picture window size. Chrome may clamp the option values if they are too large or too small to fit a user-friendly window size.
```

**Review by rachelandrew — APPROVED** [review]
With a typo edit :)

**Review by rachelandrew — APPROVED** [review]
With a typo edit :)

**Comment by beaufortfrancois** [comment]
@AaronForinton Please revert. I'll merge when Chrome implementation is in pair with Chrome spec.

**Pull request merged**